libs/blocks/global-navigation/global-navigation.css

```diff
@@ -155,7 +155,7 @@ header.global-navigation {
 /* Popup */
 .feds-popup {
   display: none;
-  padding: 12px 0;
+  padding: 0;
```

Comment:
Is any other rule declaring a padding that affects this element? It seems to me that this could be removed altogether

Reply:
True, we don't actually need that line. I removed it 👍